pkg/tnf/handlers/common/command.go

```diff
@@ -2,29 +2,19 @@ package common
 
 import "strings"
 
-// OcDebugImageID can be set by the test application that uses the test handlers which require the oc debug command to work with a specific image version
-var OcDebugImageID string
-
 const (
 	ocCommand    = "oc"
 	debugCommand = "debug"
-	imageArg     = "--image"
 )
 
 // GetOcDebugCommand returns the command base for any test handler that uses the oc debug command
 func GetOcDebugCommand() string {
```

Comment:
this whole file can probably be removed. it was introduced solely for customized image id

Reply:
Good idea, I'll check it out.